src/main/java/com/uwaterloo/connect/security/AuthenticatedUserProvider.java

```diff
@@ -0,0 +1,19 @@
+package com.uwaterloo.connect.security;
+
+import com.uwaterloo.connect.model.User;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+
+public class AuthenticatedUserProvider {
+    // Use this method to get authenticated user anywhere in the code
+    public static User getCurrentUser() {
```

Comment:
@JobanThind , we could consider just making this a bean instead of static to keep it in line with the rest of your code

Reply:
Done